docs/Developer Documentation/Logger/index.md

```diff
@@ -0,0 +1,159 @@
+# SDL Logger
+
+Open source version of SDL is using log4cxx for logging.
+Because of lack of log4cxx stability OEM should have an ability to replace log4cxx logger with any other logging boost or syslog.
```

Comment:
Please change to:
To provide flexibility SDL now provides the capability to  replace the log4cxx logger with any other logging package such as boost or syslog.

Reply:
Fixed with  
aeadebe